src/index.js

```diff
@@ -33,6 +33,16 @@ const displayTodo = () => {
       const taskName = e.target.parentElement.lastElementChild;
       if (e.target.checked) {
         taskName.style.textDecoration = 'line-through';
+        todos[e.target.dataset.id].completed = true;
+        // console.log(todos[e.target.dataset.id].completed);
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.